src/migration/1731065998183-SourceDefaultImage.ts

```diff
@@ -0,0 +1,13 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class SourceDefaultImage1731065998183 implements MigrationInterface {
+  name = 'SourceDefaultImage1731065998183'
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`ALTER TABLE "source" ALTER COLUMN "image" SET DEFAULT 'https://media.daily.dev/image/upload/s--LrHsyt2T--/f_auto/v1692632054/squad_placeholder_sfwkmj'`);
+  }
```

Comment:
I hate that we have this kind of migrations all the time :hidethepain:

Reply:
This time I blame @idoshamun. It's the same image, just different URL.